pages/gatherings/index.vue

```diff
@@ -0,0 +1,208 @@
+                </v-card-subtitle>
+              </v-col>
+              <v-col cols="auto">
+                <v-progress-circular
+                  rotate="-90"
+                  color="primary"
+                  background-color="#DADADA"
+                  size="60"
+                  class="font-weight-bold title"
+                  :value="100*gathering.collected/gathering.required"
+                >
+                  {{ Math.floor(100 * gathering.collected / gathering.required) }}%
+                </v-progress-circular>
+              </v-col>
+            </v-row>
+
+            <v-row class="mt-n10">
+              <v-col>
+                <v-card-text style="font-size: 14px;">
+                  {{ gathering.info }}
+                </v-card-text>
+              </v-col>
+            </v-row>
+
+            <v-row justify="space-around" class="mt-n8 pl-2">
+              <v-col cols="9">
+                <v-row justify="space-between">
+                  <v-col class="pb-1">
+                    <p class="subtitle mb-0 checkpoint">
+                      {{ $t('home.slideshow.collected') }}
+                    </p>
+                    <p class="font-weight-bold mb-0">
+                      {{ gathering.collected | rubles }}
+                    </p>
+                  </v-col>
+                  <v-spacer />
+                  <v-col class="pb-1">
+                    <p class="subtitle mb-0 checkpoint">
+                      {{ $t('home.slideshow.remaining') }}
+                    </p>
+                    <p class="font-weight-bold mb-0">
+                      {{ gathering.required - gathering.collected | rubles }}
+                    </p>
+                  </v-col>
+                  <v-spacer />
+                  <v-col class="pb-1 text-right">
+                    <p class="subtitle mb-0 checkpoint">
+                      {{ $t('home.slideshow.required') }}
+                    </p>
+                    <p class="font-weight-bold mb-0">
+                      {{ gathering.required | rubles }}
+                    </p>
+                  </v-col>
+                </v-row>
+                <v-progress-linear
+                  height="9"
+                  rounded
+                  background-color="#DADADA"
+                  :value="100*gathering.collected/gathering.required"
+                />
+              </v-col>
+              <v-col cols="auto" class="buttonContainer">
+                <v-btn
+                  rounded
+                  depressed
+                  dark
+                  color="#00AC00"
+                  class="text-none donate"
+                  style="font-weight: normal;"
+                >
+                  {{ $t('home.slideshow.donate') }}
+                </v-btn>
+              </v-col>
+            </v-row>
+          </v-col>
+        </v-row>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
```

Comment:
```suggestion
import Vue from 'vue';

export default Vue.extend({
```
to make use of type hints